packages/api/src/services/collateral-prices.ts

```diff
@@ -13,6 +13,18 @@ type Dependencies = {
   appClients: AppClients;
 };
 
+const tokenAddressesMap = {
+  uma: {
+    [CHAIN_IDs.mainnet]: "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828",
+    [CHAIN_IDs.polygon_matic]: "0x3066818837c5e6eD6601bd5a91B0762877A6B731",
+  },
+};
+
+const getTokenAddressFromOtherChain = (chainId: ChainId, address: string, toChainId: ChainId) => {
```

Comment:
again can be simplified if we are just assuming we want mainnet address.  i mean we dont have to assume that , and maybe its better not to, idk